python/src/scipp/compat/mantid.py

```diff
@@ -88,8 +90,10 @@ def ConvertEventWorkspaceToDataset(ws, drop_pulse_times):
     comp_info = convert_instrument(ws)
     pos, num = initPosSpectrumNo(nHist, ws)
 
+    # TODO Use unit information in workspace, if available.
```

Comment:
Can events be anything else apart from counts?

Reply:
i don't know what the units for weighted events would be

Reply:
Actually, shouldn't it be TOF (or whatever te X unit is), after all this is just the sparse coord? Weighted or not, that should not change.

The weight itself should probably have unit `counts`, since adding all the weights should give a histogram "Y", which is counts.

Reply:
yes, you're right it should be microseconds